### PR TITLE
Better friendlier messages

### PR DESCRIPTION
**Pull request by SohailHaqyar**

### CODE REVIEW

File: src/error-details/notification.ts [hunk 1]
```diff
@@ -3,84 +3,69 @@ import { IDetailsMap } from '../interfaces';
 
 export const notificationDetailsMap: IDetailsMap = {
   [Details.NotificationMissingFromLocals]: {
```
Comment (zbagley) [inline src/error-details/notification.ts:5]: Are we sure this is 100% always a ServerError?

File: src/error-details/organization.ts [hunk 2]
```diff
@@ -3,92 +3,108 @@ import { IDetailsMap } from '../interfaces';
...
+      'Organization connection does not exist. No further action required.',
     technichalReason:
       'The organization connection was not found in the database',
   },
 
   [Details.OrganizationHasNoUsers]: {
     friendlyMessage: 'Organization has no users',
-    technichalReason: 'The organization you passed in has no users',
+    technichalReason:
+      'The organization you passed in has no users. This should literally never happen, like seriously!.',
```
Comment (zbagley) [inline src/error-details/organization.ts:84]: ( ͡ᵔ ͜ʖ ͡ᵔ)

File: src/error-details/user.ts [hunk 3]
```diff
@@ -2,154 +2,172 @@ import { Details, IDetailsMap } from '..';
...
-    friendlyMessage: 'Registration access code expired',
+    friendlyMessage:
+      'Sorry, it seems like the link you are trying to use has expired. Please request a new link to register.',
     technichalReason: 'The registration access code has expired',
   },
 
   [Details.UserActiveRegistrationMissing]: {
-    friendlyMessage: 'User is not in active registration',
+    friendlyMessage:
+      'Sorry, it seems like your account needs to be in the proccess of registration to perform this action. Please complete the registration process to proceed.',
```
Comment (zbagley) [inline src/error-details/user.ts:127]: The account associated with this activation link does not exist. Please complete the registration process again.

File: src/error-details/user.ts [hunk 4]
```diff
@@ -2,154 +2,172 @@ import { Details, IDetailsMap } from '..';
...
     technichalReason:
       'The user.registration.activeRegistration was not found in the database',
   },
 
   [Details.UserNotInRequestedOrganization]: {
-    friendlyMessage: 'User is not in your organization',
+    friendlyMessage:
+      'This action requires you to be a member of the requested organization to perform this action. ',
     technichalReason: 'The user does not belong to the requested organization',
```
Comment (zbagley) [inline src/error-details/user.ts:135]: We may need to change this one depending on occurrences.

File: src/error-details/user.ts [hunk 5]
```diff
@@ -2,154 +2,172 @@ import { Details, IDetailsMap } from '..';
...
-    friendlyMessage: 'You must be an integrator',
+    friendlyMessage:
+      'The action requires you to be a member of an Integrator organization.',
     technichalReason: 'The user organization type is not integrator',
   },
 
   [Details.UserNotEndUser]: {
-    friendlyMessage: 'You must be an end user',
+    friendlyMessage:
+      'The action requires you to be a member of an Integrator organization. Either create an organization or join one to enable this action',
```
Comment (zbagley) [inline src/error-details/user.ts:158]: The action requires you to be a member of an End User organization. (also delete 2nd sentence)

Comment (SohailHaqyar) [reply src/error-details/user.ts]: https://github.com/SalientSys/salient-sst-errors/pull/16/commits/8c02061f2c86e856c099a6341115b995e2b31dd5

File: src/error-details/user.ts (hunk 4 above)
Comment (SohailHaqyar) [reply src/error-details/user.ts]: https://github.com/SalientSys/salient-sst-errors/pull/16/commits/8c02061f2c86e856c099a6341115b995e2b31dd5

File: src/error-details/user.ts (hunk 3 above)
Comment (SohailHaqyar) [reply src/error-details/user.ts]: https://github.com/SalientSys/salient-sst-errors/pull/16/commits/8c02061f2c86e856c099a6341115b995e2b31dd5

File: src/error-details/user.ts [hunk 6]
```diff
@@ -2,154 +2,172 @@ import { Details, IDetailsMap } from '..';
...
+    friendlyMessage:
+      'There was an error processing your request. Please Contact support for assitance.',
     technichalReason:
       'The access code did not match the access code in the database',
   },
 
   [Details.RegistrationAccessCodeExpired]: {
-    friendlyMessage: 'Registration access code expired',
+    friendlyMessage:
+      'Sorry, it seems like the link you are trying to use has expired. Please request a new link to register.',
```
[the remark this answers is not in the document]
Comment (SohailHaqyar) [reply src/error-details/user.ts]: https://github.com/SalientSys/salient-sst-errors/pull/16/commits/8c02061f2c86e856c099a6341115b995e2b31dd5

File: src/error-details/notification.ts (hunk 1 above)
Comment (SohailHaqyar) [reply src/error-details/notification.ts:5]: https://github.com/SalientSys/salient-sst-errors/pull/16/commits/8c02061f2c86e856c099a6341115b995e2b31dd5

File: src/error-details/deployments.ts [hunk 7]
```diff
@@ -3,114 +3,132 @@ import { IDetailsMap } from '../interfaces';
...
-    friendlyMessage: 'Deployment id missing from body',
+    friendlyMessage:
+      'A Critical Server error has occurred. Please contact support.',
     technichalReason:
       'The deployment id was not found in neither the req.body.deploymentId nor req.body.managementServerGuid',
   },
   [Details.DeploymentAlreadyRegisteredInDifferentOrg]: {
-    friendlyMessage: 'Deployment already registered in different organization',
+    friendlyMessage:
+      'This deployment is already registered in different organization. Please disconnect the deployment and try again.',
```
[the remark this answers is not in the document]
Comment (SohailHaqyar) [reply src/error-details/deployments.ts]: https://github.com/SalientSys/salient-sst-errors/pull/16/commits/8c02061f2c86e856c099a6341115b995e2b31dd5

**Review by zbagley — APPROVED** [review]

**Pull request merged**